broadinstitute/pearl
# [JN-455] basic participant notes

This adds basic text note creation to the admin tool.  Notes are not yet editable.  Later work might include:
1. markdown for notes
2. editing or deleting notes (and maintaining audit trail)
3. Refining the "Profile & Notes" enrollee view screen to be more compact
4. UX for adding notes associated with kits (the backend for supporting that is in place, but no UX)

Notable additions support of this feature include:
1.  adding a new provider to load admin users so that usernames could be associated with notes. Eventually, there'll be a whole bunch of "background stuff to load on sign-in" like other usernames, kit types, etc... and we'll consolidate that into a single endpoint
2. Added a "TimeShiftable" interface for entities that can be populated in the past.

TO TEST:
1. redeploy AdminApiApp, repopulate ourhealth
2. go to https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/participants/OHSALK. 
3. confirm you see two notes at the bottom of the page
![image](https://github.com/broadinstitute/juniper/assets/2800795/cd739d99-a846-4fd0-94c4-ac120c4c10bc)
4. Add a new note
5. reload page, confirm note persists
